recipes/poolsnp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.0.0" %}
+
+package: 
+  name: poolsnp
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/capoony/PoolSNP.git
```

Comment:
Could you please ask the authors for a release/tag ?

Reply:
[Asked the authors for a release](https://github.com/capoony/PoolSNP/issues/10) - Awaiting response, else I will maintain a fork of the repo with releases